_layouts/page.html

```diff
@@ -11,14 +11,14 @@
     {% include header.html %}
   </head>
   <body class="workshop">
-    <div class="container card">
```

Comment:
In the new CSS, .card denotes a white background, and I think removing it will break the style. Was it removed intentionally?